app/models/hubstats/pull_request.rb

```diff
@@ -23,7 +24,8 @@ class PullRequest < ActiveRecord::Base
 
     belongs_to :user
     belongs_to :repo
-    has_and_belongs_to_many :labels, :join_table => 'hubstats_labels_pull_requests'
+    belongs_to :deploy
+    has_and_belongs_to_many :labels, :join_table => "hubstats_labels_pull_requests"
```

Comment:
Yes. The label is still attached to the pull request, so saying that a pull request also belongs to a deploy doesn't affect the labels.